dockerfiles/theia/src/extensions.json

```diff
@@ -0,0 +1,16 @@
+{
+    "extensions": [
+        {
+            "name": "theia-machines-extension",
+            "source": "https://github.com/eclipse/che-theia-machines-plugin.git",
+            "version": "0.3.12",
```

Comment:
Please rename to `version` to `branch` ot whatever because it is actually checkout argument.